src/lib.rs

```diff
@@ -41,10 +31,8 @@ impl Drone {
     fn send(&self, raw_frame: Vec<u8>) -> AnyResult<()> {
         self.sender.send(raw_frame).map_err(AnyError::new)
     }
-}
 
-impl Drone {
-    fn sequence_id(&self, buffer_id: frame::BufferID) -> u8 {
+    pub fn sequence_id(&self, buffer_id: frame::BufferID) -> u8 {
```

Comment:
I tried to keep sequence_id private, to avoid having people call it when they shouldn't. is it now required because of the for_drone function ?

Reply:
we can make it `pub (crate)`?! Idk if it works for fns but I will try!